e3sm_diags/driver/aerosol_aeronet_driver.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import print_function
+
+import os
```

Comment:
```suggestion
import os
```
We don't need to import ``__future__`` since that library is meant to bridge some Python 3 capabilities with Python 2.

Per https://stackoverflow.com/questions/32032697/why-does-using-from-future-import-print-function-breaks-python2-style-print:

>That's the whole point of from future import print_function; to bring the print function from Python 3 into Python 2.6+.